.circleci/config.yml

```diff
@@ -1,231 +1,143 @@
+      - run:
+          name: set TESTKIT_ENABLE_ZIP for artifacts on linux
+          command: echo "export TESTKIT_ENABLE_ZIP=true" >> $BASH_ENV
       - run:
-          when: on_success
-          name: All Good! Close CTC case/implementations
+          name: Nuts
           command: |
-            if [ -z "${SF_CHANGE_CASE_SFDX_AUTH_URL}" ] || [ -z "${SF_CHANGE_CASE_TEMPLATE_ID}" ] || [ -z "${SF_CHANGE_CASE_SCHEDULE_BUILD}" ]; then
-                echo "Environment not configured for CTC"
-            else
-                sfchangecase close --location "https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
-            fi
+            echo "Using node: $(node --version)"
+            echo "Environment Variables:"
+            env
+            NODE_OPTIONS=--max-old-space-size=8192 yarn test:nuts
 
 workflows:
-  version: 2.1
-
-  commit-workflow:
-    unless: << pipeline.parameters.publish >>
+  version: 2
+  test-and-release:
     jobs:
-      - node-latest
-      - node-12
-      - cache:
+      - release-management/validate-pr:
           filters:
-            tags:
-              only: /^v.*/
             branches:
-              ignore: /.*/
-      - run-win-tests
-
-  publish-workflow:
-      when: << pipeline.parameters.publish >>
-      jobs:
-        - slack/approval-notification:
-            message: Pending Approval for Publish of SDRL (Source Deploy Retrieve Library)
-            channel: 'pdt_releases'
-            color: '#0E1111'
-        - node-latest
-        - node-12
-        - run-win-tests
-        - hold: # Requires manual approval in Circle Ci
-            type: approval
-        - publish:
-            context:
-              - pdt-publish-restricted-context
-              - CLI_CTC
-            filters:
-              branches:
-                only:
-                  - main
-            requires:
-              - node-latest
-              - node-12
-              - run-win-tests
-              - hold
+              ignore: main
+      - release-management/test-package:
+          matrix:
+            parameters:
+              os:
+                - linux
+              node_version:
+                - latest
+                - lts
+                - maintenance
+            exclude:
+              - os: windows
+                node_version: lts
+              - os: windows
+                node_version: maintenance
+      - external-nut:
+          filters:
+            branches:
+              # we don't run again on main because
+              # 1) we already ran on a branch
+              # 2) they aren't required and would run in parallel to release job
+              ignore: main
+          requires:
+            - release-management/test-package
+          sfdx_version: latest
+          size: 2xlarge
+          matrix:
+            parameters:
+              os: [linux]
+              node_version: [lts]
+              external_project_git_url:
```

Comment:
do we not have `org:create` NUTs for toolbelt, that deploy to the org?

Reply:
we do, and I wanted to do that, but there's some more complicated SSH setup keys that I don't understand for circle to be able to clone a git private repo that's not the project's repo.

I had the same problem on sfdx-core and haven't solved it yet.